### ISSUE TITLE
Feature Proposal: Alt text customization/photo credits

### ISSUE THREAD

**Issue by heavyelement** [bonfire-networks/bonfire-app#2161]
**Is your feature request related to a problem? Please describe.**
I have a photographer friend who wants to donate some professional photography for use on our instance. Specifically for the Branding -> "Left panel background image" and the General Configuration -> "Background image." I'd like to be able to give her credit for her work by linking to either her profile on the site or her website. Additionally, the ability to customize alt text to these images would help give the creator exposure and help surface our instance in search results.

**Describe the solution you'd like**
I'd like to see two things: anywhere there's a background image upload field, an optional "Alt text" field and optional Markdown sensitive "Photo credit" field should be rendered. Either through a popover modal (preferred) or as block-level fields.

EDIT: The photo credit contents should be rendered Markdown *over* the image in the bottom left corner of the respective image (probably in a `<figcaption>` tag).

**Describe alternatives you've considered**
For the background photo, we can add a photo credit in the About -> "Welcome Description"
For the login screen, we can add a photo credit in the About -> "Welcome Description" but if they're different images from different photographers, this isn't an ideal workaround since credit would need to be given to both regardless of which image is displayed.

Neither of these solutions control us over the alt text for accessibility/search engine reach.
